src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -230,14 +230,16 @@ private function handleException(\Exception $e, $request, $type)
         // a listener might have replaced the exception
         $e = $event->getException();
 
-        if (!$event->hasResponse()) {
+        if ($event->hasResponse()) {
+            $response = $event->getResponse();
+        } elseif ($e instanceof HttpExceptionInterface) {
+            $response = new Response($e->getMessage());
```

Comment:
Wondering... couldn't `$e->getMessage()` potentially expose something that is not meant to be displayed to a user ever?

Maybe better use `Response::$statusTexts[$code]`?

Reply:
When twig is installed, this is the message that is displayed, so I don't think there is anything harmful here.

Reply:
Hmm really? I thought with twig we only show the `status_text`?

https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php#L66

Reply:
correct, fixed